### DISCUSSION TITLE
why only install x64-linux when I install android library.

### DISCUSSION THREAD

**Discussion by PeacefulWindy** [microsoft/vcpkg#29355] [Q&A] [answered]
system: Ubuntu 22.04.1 LTS in WSL2
vcpkg git log: commit 91dd61bd441a68b4017b61011d0350b2e6aeeccf (HEAD -> master, origin/master, origin/HEAD)

question:
when i use command: "./vcpkg install --triplet x64-android",vcpkg only install x64-linux library but not x64-android.
shell msg:
`Detecting compiler hash for triplet x64-linux...`

this is vcpkg.json
`
{
  "name": "myproject",
  "version-semver": "1.0.0",
  "dependencies": [
    {
      "name": "angle",
      "host":true,
      "version>=": "chromium_4472#7"
    },
    {
      "name":"assimp",
      "host":true,
      "version>=":"5.2.5"
    },
    {
      "name": "brotli",
      "host":true,
      "version>=": "1.0.9#4"
    },
    {
      "name": "bzip2",
      "host":true,
      "version>=": "1.0.8#3"
    },
    {
      "name": "egl-registry",
      "host":true,
      "version>=": "2022-09-20"
    },
    {
      "name": "egl",
      "host":true,
      "version>=": "2022-12-04"
    },
    {
      "name": "ffmpeg",
      "host":true,
      "version>=": "4.4.3#3"
    },
    {
      "name": "fmt",
      "host":true,
      "version>=":"9.1.0#1"
    },
    {
      "name": "freetype",
      "host":true,
      "version>=": "2.12.1#3"
    },
    {
      "name": "glad",
      "features": [
        "extensions",
        "loader",
        "gl-api-latest",
        "gles2-api-latest"
      ],
      "host":true,
      "version>=": "0.1.36"
    },
    {
      "name": "glfw3",
      "host":true,
      "version>=": "3.3.8#2"
    },
    {
      "name": "glm",
      "host":true,
      "version>=": "0.9.9.8#1"
    },
    {
      "name": "icu",
      "host":true,
      "version>=": "72.1"
    },
    {
      "name": "libmariadb",
      "host":true,
      "version>=": "3.3.1#2"
    },
    {
      "name": "libpng",
      "host":true,
      "version>=": "1.6.39"
    },
    {
      "name": "libzip",
      "host":true,
      "version>=": "1.9.2"
    },
    {
      "name": "lua",
      "host":true,
      "version>=": "5.4.4#3"
    },
    {
      "name": "nlohmann-json",
      "host":true,
      "version>=": "3.11.2"
    },
    {
      "name": "opengl-registry",
      "host":true,
      "version>=": "2022-09-29"
    },
    {
      "name": "opengl",
      "host":true,
      "version>=": "2022-12-04"
    },
    {
      "name": "openssl",
      "host":true,
      "version>=": "3.0.7#2"
    },
    {
      "name": "physx",
      "host":true,
      "version>=": "4.1.2#5"
    },
    {
      "name":"refl-cpp",
      "host":true,
      "version>=":"0.12.3#1"
    },
    {
      "name": "sol2",
      "host":true,
      "version>=": "3.3.0"
    },
    {
      "name": "zlib",
      "host":true,
      "version>=": "1.2.13"
    }
  ],
  "features":
  {
    "angle":
    {
      "description":"angle",
      "supports": "!x86"
    },
    "assimp":
    {
      "description":"assimp",
      "supports": "!x86"
    },
    "brotli":
    {
      "description":"brotli",
      "supports": "!x86"
    },
    "bzip2":
    {
      "description":"bzip2",
      "supports": "!x86"
    },
    "egl-registry":
    {
      "description":"egl-registry",
      "supports": "!x86"
    },
    "egl":
    {
      "description":"egl",
      "supports": "!x86"
    },
    "ffmpeg":
    {
      "description":"ffmpeg",
      "supports": "!x86"
    },
    "fmt":
    {
      "description":"fmt",
      "supports": "!x86"
    },
    "freetype":
    {
      "description":"freetype",
      "supports": "!x86"
    },
    "glad":
    {
      "description":"glad",
      "supports": "!x86"
    },
    "glfw3":
    {
      "description":"glfw3",
      "supports": "!x86 & !android"
    },
    "glm":
    {
      "description":"glm",
      "supports": "!x86"
    },
    "icu":
    {
      "description":"icu",
      "supports": "!x86"
    },
    "libmariadb":
    {
      "description":"libmariadb",
      "supports": "!x86"
    },
    "libpng":
    {
      "description":"libpng",
      "supports": "!x86"
    },
    "libzip":
    {
      "description":"libzip",
      "supports": "!x86"
    },
    "lua":
    {
      "description":"libzip",
      "supports": "!x86"
    },
    "nlohmann-json":
    {
      "description":"libzip",
      "supports": "!x86"
    },
    "opengl-registry":
    {
      "description":"libzip",
      "supports": "!x86"
    },
    "opengl":
    {
      "description":"opengl",
      "supports": "!x86"
    },
    "openssl":
    {
      "description":"openssl",
      "supports": "!x86"
    },
    "physx":
    {
      "description":"physx",
      "supports": "!x86"
    },
    "refl-cpp":
    {
      "description":"refl-cpp",
      "supports": "!x86"
    },
    "sol2":
    {
      "description":"sol2",
      "supports": "!x86"
    },
    "zlib":
    {
      "description":"zlib",
      "supports": "!x86"
    }
  },
  "overrides":[
    {
        "name":"ffmpeg",
        "version": "4.4.3#3"
    }
  ],
  "builtin-baseline":"e51afe4695ebf5e6c6f523dbd3bb518db1970531",
  "supports" : "android & windows & linux & x64 & arm64"
}
`

**Comment by autoantwort** [comment, marked as answer]
Because you say `host: true` and your host system is `x64-linux`

**Reply by PeacefulWindy** [reply]
you're right,thank you!(＞﹏＜)
